service.subtitles.supersubtitles/addon.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="service.subtitles.supersubtitles"
+       name="Super Subtitles"
+       version="0.0.23"
+       provider-name="fape">
+  <requires>
+    <import addon="xbmc.python" version="2.26.0"/>
+    <import addon="xbmc.addon" version="17.9.910"/>
+    <import addon="script.module.kodi-six" version="0.1.0"/>
+    <import addon="vfs.libarchive" version="1.0.5"/>
+  </requires>
+  <extension point="xbmc.subtitle.module"
+             library="service.py" />
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">Super Subtitles</summary>
+    <description lang="en_GB">Search and download subtitles from Super Subtitles (feliratok.info)</description>
+    <platform>all</platform>
+    <license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
+    <forum>https://forum.kodi.tv/showthread.php?tid=188591</forum>
+    <website>https://www.feliratok.info/index.php</website>
+    <email>fape88 at gmail dot com</email>
+    <language></language>
```

Comment:
please remove the empty language tag

Reply:
There are 8 addons in leai branch with empty language tag.

Reply:
[What about offical documentation](https://kodi.wiki/view/Addon.xml#.3Clanguage.3E):

> When there is no specific language provided in your content, leave it blank.

Reply:
Subtitle addons do not provide content.